src/block/tile/StructureBlock.php

```diff
@@ -0,0 +1,376 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\block\tile;
+
+use pocketmine\block\utils\StructureAnimationMode;
+use pocketmine\block\utils\StructureAxes;
+use pocketmine\block\utils\StructureBlockType;
+use pocketmine\block\utils\StructureRotation;
+use pocketmine\math\Vector3;
+use pocketmine\nbt\NbtDataException;
+use pocketmine\nbt\tag\CompoundTag;
+use pocketmine\network\mcpe\protocol\StructureBlockUpdatePacket;
+use pocketmine\world\World;
+
+class StructureBlock extends Spawnable{
+	public const TAG_ANIMATION_MODE = "animationMode";
+	public const TAG_ANIMATION_SECONDS = "animationSeconds";
+	public const TAG_DATA = "data";
+	public const TAG_DATA_FIELD = "dataField";
+	public const TAG_IGNORE_ENTITIES = "ignoreEntities";
+	public const TAG_INTEGRITY = "integrity";
+	public const TAG_IS_POWERED = "isPowered";
+	public const TAG_MIRROR = "mirror";
+	public const TAG_REDSTONE_SAVE_MODE = "redstoneSaveMode";
+	public const TAG_REMOVE_BLOCKS = "removeBlocks";
+	public const TAG_ROTATION = "rotation";
+	public const TAG_SEED = "seed";
+	public const TAG_SHOW_BOUNDING_BOX = "showBoundingBox";
+	public const TAG_STRUCTURE_NAME = "structureName";
+	public const TAG_X_STRUCTURE_OFFSET = "xStructureOffset";
+	public const TAG_Y_STRUCTURE_OFFSET = "yStructureOffset";
+	public const TAG_Z_STRUCTURE_OFFSET = "zStructureOffset";
+	public const TAG_X_STRUCTURE_SIZE = "xStructureSize";
+	public const TAG_Y_STRUCTURE_SIZE = "yStructureSize";
+	public const TAG_Z_STRUCTURE_SIZE = "zStructureSize";
```

Comment:
To go along with the other tiles, you need to comment the Tag type (e.g. TAG_Byte, ...)